manifests/init.pp

```diff
@@ -360,6 +365,7 @@
   Boolean $manage_etcd                         = true,
   Optional[String] $kube_api_advertise_address = undef,
   Optional[String] $etcd_version               = '3.1.12',
+  Optional[String] $etcd_hostname              = $hostname,
```

Comment:
Hi @sw0x2A,

Could you please change the hostname to use the new syntax for the facts, similar to line 350 ? 

Thanks.

Reply:
@carabasdaniel Done.

Reply:
If you want the etcd hostname to be FQDN the variable should be $facts['fqdn']

Reply:
I want to be able to set the etcd instance name to $fqdn and therefore I introduced a parameter for it in this PR. My intention is not to change the default. Instance name is currently set to $hostname in the template of etcd config file which is fine for most users of this module and changing the default might even be a breaking change.